$NetBSD: patch-src_qt-console_bat.pro.in,v 1.1 2012/10/06 17:54:53 christos Exp $

- comment out the confs paths and command
- use just the compiler not libtool for QMAKE_LINK

--- src/qt-console/bat.pro.in.orig	2012-09-13 04:51:19.000000000 -0400
+++ src/qt-console/bat.pro.in	2012-10-05 11:17:23.000000000 -0400
@@ -10,8 +10,8 @@
 
 bins.path = /$(DESTDIR)@sbindir@
 bins.files = bat
-confs.path = /$(DESTDIR)@sysconfdir@
-confs.commands = ./install_conf_file
+#confs.path = /$(DESTDIR)@sysconfdir@
+#confs.commands = ./install_conf_file
 help.path = /$(DESTDIR)@docdir@
 help.files = help/*.html images/status.png images/mail-message-new.png
 
@@ -22,7 +22,7 @@
 LIBS        += -L../lib -lbaccfg -lbac -L../findlib -lbacfind @OPENSSL_LIBS@
 LIBTOOL_LINK = @QMAKE_LIBTOOL@ --silent --tag=CXX --mode=link
 LIBTOOL_INSTALL = @QMAKE_LIBTOOL@ --silent --mode=install
-QMAKE_LINK   = $${LIBTOOL_LINK} $(CXX)
+QMAKE_LINK   = $(CXX)
 QMAKE_INSTALL_PROGRAM = $${LIBTOOL_INSTALL} install -m @SBINPERM@ -p
 QMAKE_CLEAN += .libs/* bat
 
